modules/java_api/org/intel/openvino/IECore.java

```diff
@@ -38,61 +40,54 @@ private static String getLibraryName(String name, String linux_ver) {
     // Use this method to initialize native libraries and other files like
     // plugins.xml and *.mvcmd in case of the JAR package os OpenVINO.
     public static void loadNativeLibs() {
-        final String[] nativeFiles = {
-            "plugins.xml",
-            "usb-ma2x8x.mvcmd",
-            "pcie-ma2x8x.mvcmd",
-            "cache.json",
+        // A set of required libraries which are listed in dependency order.
+        final String[] nativeLibs = {
             "tbb",
             "tbbmalloc",
             "ngraph",
-            "gna",
             "inference_engine_transformations",
             "inference_engine",
-            "inference_engine_ir_reader",
```

Comment:
Are you sure we don't need it?

Reply:
how do we decide which library should be in this list? At least, add comments for newcomers..

Reply:
There are tests that show we dont't need to load plugin libraries (System.load) - the libraries which are linked by dlopen. We read IR in tests without errors.

Reply:
In this case I suppose you don't need to have to keep the following dependencies in this list:
```
onnx_importer
inference_engine_legacy
inference_engine_lp_transformations
```
Because they are plugin dependences and loaded together with plugins

Reply:
@ilya-lavrenov, That's good point but without `System.load` we couldn't use MKLDNNPlugin. I guess that it works in the same way as we split the libraries to `.dylib` and `.so`. So in Java, it's required to `System.load` all the libraries which are `.dylib` in Mac. I think it make sense to add Windows tests to check it more precisely.

Reply:
ONNX reader is not compiled for contrib so I tested it and removed in the final PR revision.

We have this list of libraries because they are dependencies but not a plugins (dlopen):

* `onnx_importer` -> `inference_engine_onnx_reader`
* `inference_engine_legacy` and `inference_engine_lp_transformations` -> `MKLDNNPlugin`

On Linux, they are not linked by dlopen but should be loaded by `System.load`